### ISSUE TITLE
RxFormBuilder does not recognize Date default value

### ISSUE THREAD

**Issue by AliYusuf95** [rxweb/rxweb#269]
# Describe the bug
During parsing of the data object with a Date value field if it's inside an array, RxFormBuilder throws an exception `ERROR Error: Cannot read property 'conditionalValidationProps' of undefined`.

Related issue: https://github.com/rxweb/rxweb/issues/148

# Expected Behavior
Date field should be recognized add added to RxFormGroup as RxFormControl.

# To Reproduce
```ts
const form = this._formBuilder.group({
        date: [new Date(), [RxwebValidators.required()]],
});
```

# Url (stackblitz or plunker or other)
https://stackblitz.com/edit/angular-gg8jos-1r4nve?embed=1&file=src/app/group-complete.component.ts

# Package Version
@rxweb/reactive-form-validators@1.9.7

**Comment by ajayojha** [comment]
Thank You for reporting this issue.

**Comment by ajayojha** [comment]
@AliYusuf95 The has been resolved in 1.9.8 release, please update the package.
